modules/tinymce/src/plugins/autosave/main/ts/core/Time.ts

```diff
@@ -5,16 +5,15 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-const parse = (timeString: string | undefined, defaultTime: string): number => {
+const parse = (timeString: string): number => {
   const multiples: Record<string, number> = {
     s: 1000,
     m: 60000
   };
-  const toParse = (timeString || defaultTime);
 
-  const parsedTime = /^(\d+)([ms]?)$/.exec('' + toParse);
+  const parsedTime = /^(\d+)([ms]?)$/.exec(timeString);
```

Comment:
Should we test whether this regex fails, and pass it as an error message back up to the option validator?

Reply:
Out of scope for these changes.